### PR TITLE
New (2024-2025 Website maintenance)

### PR DESCRIPTION
**Pull request by Olirva-M**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "version": "1.0.0",
+  "description": "Welcome to the official repository for the ACM Student Chapter website at [SSN College of Engineering](https://www.ssn.edu.in)! Here, you will find all the source code and assets for our website, which is dedicated to promoting computer science and technology education and opportunities for students at SSN College.",
+  "main": "form-submission-handler.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
+}
```
Comment (siriscmv) [inline package.json:12]: Is this file needed? I dont think it does anything

Comment (Olirva-M) [reply package.json:12]: Its not needed. I forgot to delete.

Comment (siriscmv) [reply package.json:12]: can you delete it?

Comment (Olirva-M) [reply package.json]: Done
